StringArt/StringArt.js

```diff
@@ -0,0 +1,128 @@
+import { Polar } from "../sketchlib/Polar.js";
+
+const MODULUS = 60;
+
+/**
+ * Polynomial ax^3 + bx^2 + cx + d where the coefficients
+ * and values are evaluated mod n
+ */
+class Polynomial {
+  constructor(a, b, c, d) {
+    this.coefficients = [a, b, c, d];
+  }
+
+  compute(x) {
+    const x2 = (x * x) % MODULUS;
+    const x3 = (x2 * x) % MODULUS;
+    const [a, b, c, d] = this.coefficients;
+
+    return (a * x3 + b * x2 + c * x + d) % MODULUS;
+  }
+
+  to_string() {
```

Comment:
Test cases:

- 0 coefficients should produce "0 (mod N)"
- individual coefficients should only render a single term
- All the coefficients should be joined by + signs